kratos/includes/fill_communicator.h

```diff
@@ -102,6 +114,22 @@ class KRATOS_API(KRATOS_CORE) FillCommunicator
      */
     virtual void Execute();
 
+    /**
+     * @brief Function to bring entities from other partitions
+     * @details This function is intended to bring entities from other partitions. The map indicates the partitions to bring and the vector the entities to bring from each partition. In the current serial case it does nothing.
+     * @note For the parallel implementation see ParallelFillCommunicator.
+     * @param rNodesToBring Nodes to bring from other partitions
+     * @param rElementsToBring Elements to bring from other partitions
+     * @param rConditionsToBring Conditions to bring from other partitions
+     * @param CallExecuteAfterBringingEntities Call Execute after bringing entities
+     */
+    virtual void BringEntitiesFromOtherPartitions(
```

Comment:
> `BringEntitiesFromOtherPartitions`

sooo, `Gather`? The rest of the signature should be self-explanatory.

Reply:
Ikay

Reply:
I meant replace the entire name `BringEntitiesFromOtherPartitions` with `Gather`, or is that too short?

Reply:
GatherFromOtherPartitions? (to me just Gather is too short)